### DISCUSSION TITLE
MudBlazor Wrapped Oqtane Razor Component Library

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#3841] [General]
I've embarked on an ambitious project dubbed MORC (Mud-Oqtane Razor Components), aimed at enriching the experience of Mud-ified Modules and Themes. One of the pivotal challenges I'm currently tackling revolves around the integration of Oqtane controls. These controls, while potent within the framework, present obstacles as they cannot be readily inherited from or manipulated outside its confines.

Though still in its nascent stages, I'm wholeheartedly welcoming contributions from interested parties. If you're keen to get involved, you can find the project repository here: https://github.com/leigh-pointer/MudOqtaneRazorControls

**Comment by markdav-is** [comment]
So glad you are working in the MudBlazor space.  I'm doing theming using those control and would be happy to contribute.  I've re-written a few Oqtane controls using MudBlazor and I'll push them up to my repo soon.  For the most part, we can inherit from the controls base-class for functionality, so that's handy.  We may need to contribute-back to make things customizable.  Admin Pane comes to mind.  That's not as configurable as would be handy.

**Reply by leigh-pointer** [reply]
@markdav-is When you start looking at using this extension packages it is all well and good then, the UI controlling. Nightmare.  Anyway, the repo is coming alone.  As you said some things can be inherited from and a lot not.

**Reply by markdav-is** [reply]
I'm behind on this project, but this is the repo:  https://github.com/DarkMatterOregon/maqui

**Reply by leigh-pointer** [reply]
Ok i will have a look.
